packages/core/src/shared/remoteSession.ts

```diff
@@ -251,3 +251,23 @@ export async function getDeniedSsmActions(client: IamClient, roleArn: string): P
 
     return deniedActions
 }
+
+/**
+ * Creates a new {@link ChildProcess} class bound to a specific dev environment. All instances of this
```

Comment:
Good idea extracting this 👍 

```suggestion
 * Creates a new {@link ChildProcess} class bound to a specific remote environment. All instances of this
```